eosc/cmd/getAccount.go

```diff
@@ -1,33 +1,250 @@
 package cmd
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
+	"strconv"
+	"strings"
+	"text/tabwriter"
 
+	"github.com/ryanuber/columnize"
+
+	eos "github.com/eoscanada/eos-go"
 	"github.com/spf13/cobra"
 )
 
 var getAccountCmd = &cobra.Command{
 	Use:   "account [account name]",
 	Short: "retrieve account information for a given name",
-	Args:  cobra.ExactArgs(1),
+	Long:  "retrieve account information for a given name.  For a json dump, append the argument --json.",
+	Args:  cobra.RangeArgs(1, 2),
 	Run: func(cmd *cobra.Command, args []string) {
 		api := getAPI()
 
 		accountName := toAccount(args[0], "account name")
 		account, err := api.GetAccount(accountName)
 		errorCheck("get account", err)
 
-		data, err := json.MarshalIndent(account, "", "  ")
-		errorCheck("json marshal", err)
-
-		// TODO: properly display all account details, and fetch a few
-		// other things also, to make it a complete picture (like all
-		// token balances on eosio.token ?), other tokens ?
-		fmt.Println(string(data))
+		if printJSON, _ := cmd.Flags().GetBool("json"); printJSON == true {
```

Comment:
if you check the other commands, we wrap everything in `viper.GetBool()` calls, so we can have environment variables change flags.

Also, I guess `...; printJSON {` will suffice for the bool check.

Reply:
/nonblocking

Reply:
Will update this... my bad :)